openstack_controller/datadog_checks/openstack_controller/api.py

```diff
@@ -72,6 +83,158 @@ def get_networks(self):
         raise NotImplementedError()
 
 
+class OpenstackSdkApi(AbstractApi):
```

Comment:
Could we rename this `OpenstackSDKApi`. Not a big deal tho, completely opinionated and definitely coming from my OCD issues 😄

Reply:
Says the one who wrote `Api` :trollface: